client/analytics/report/revenue/table.js

```diff
@@ -6,7 +6,7 @@ import { Component } from '@wordpress/element';
 import { format as formatDate } from '@wordpress/date';
 import { withSelect } from '@wordpress/data';
 import { compose } from '@wordpress/compose';
-import { get } from 'lodash';
+import { get, memoize } from 'lodash';
```

Comment:
We do have `memoize-one` which we could optionally move to, but there are still many places `lodash` is used, so I'm not opposed to saving this for a follow-up when we're ready to tackle that.

Reply:
Unfortunately, `memoize-one` isn't a direct replacement and works fundamentally differently than most memoization libraries - it will only return cached values if subsequent calls receive the same arguments (`===` comparison) and there is no way to customize the cache "key" like I'm doing with Lodash in this PR.